tests/verific/clocking.ys

```diff
@@ -0,0 +1,12 @@
+read -sv <<EOT
+module test(input clk, foo);
+always @(*) assert(foo);
+endmodule
+EOT
+
+verific -import test
+prep
+
+select -assert-count 1 t:$assert
+select -assert-count 1 t:$not
+select -assert-none t:$assert t:$not %% t:* %D
```

Comment:
I think we only want to test that the $assert cell exists, we don't want to constrain the specific implementation choice of checker logic beyond that, so I would remove lines 11-12 from the test.